.github/workflows/cli-test.yml

```diff
@@ -69,17 +69,17 @@ jobs:
         run: pnpm webstudio-cli build --template netlify-functions --preview && pnpm prettier --write ./app/ ./package.json
 
         # Testing fixtures for netlify-edge-functions template
-      - name: Test cli link command for netlify-edge-functions
-        working-directory: ./fixtures/webstudio-remix-netlify-edge-functions
+      - name: Test cli link command for netlify
```

Comment:
1st lets move this into fixture  packages
i.e.
```json
"scripts": {
    "fixture:link": "webstudio-cli link --link 'https://webstudio-builder-git-main-webstudio-is.vercel.app/builder/d845c167-ea07-4875-b08d-83e97c09dcce?authToken=e9d1343f-9298-4fd3-a66e-f89a5af2dd93",
    "fixture:sync": "blabla",
    "fixture:build": "blablablabla",
...
}
    ```

then using pnpm just run from the root 
```bash
pnpm --filter='./fixtures/*' run --parallel fixtures:link
pnpm --filter='./fixtures/*' run --parallel fixtures:sync
pnpm --filter='./fixtures/*' run --parallel fixtures:build
```

to never touch test.yaml again